README.md

```diff
@@ -8,18 +8,23 @@ These instructions will get you a copy of the project up and running on your loc
 
 ### Prerequisites
 
-Docker needs to be installed on your system because local deployment uses containerisation to standardise deployments across machines. For further information please read Docker docs. https://docs.docker.com/.
+Docker needs to be installed on your system because local deployment uses containerisation to standardise deployments across machines. For further information please read [Docker docs](https://docs.docker.com/).
 
-For Mac users, proceed to https://docs.docker.com/docker-for-mac/install/ and follow the instructions there. From here on, all terminal commands are written for MacOS and a zsh terminal. Make changes as necessary.
+For Mac users, proceed to [Docker for Mac](https://docs.docker.com/docker-for-mac/install/) and follow the instructions there. From here on, all terminal commands are written for MacOS and a zsh terminal. Make changes as necessary.
 
-For Windows users, proceed to https://docs.docker.com/docker-for-windows/install/ and follow the instructions there. Be aware that if you have a Windows Home account with a Windows Subsystem for Linux, then additional steps need to be taken before downloading Docker, of which you can find here https://medium.com/@sebagomez/installing-the-docker-client-on-ubuntus-windows-subsystem-for-linux-612b392a44c4.
+For Windows users, proceed to [Docker for Windows](https://docs.docker.com/docker-for-windows/install/) and follow the instructions there. Be aware that if you have a Windows Home account with a Windows Subsystem for Linux, then additional steps need to be taken before downloading [Docker](https://medium.com/@sebagomez/installing-the-docker-client-on-ubuntus-windows-subsystem-for-linux-612b392a44c4).
```

Comment:
I'd say just leave it at the first sentence and cut out "Be aware that... " because that second half isn't necessarily true now with WSL 2.

Reply:
Good to know.